projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -275,61 +257,16 @@ bootstrapApplication(AppComponent, {
 
 <div class="alert is-important">
 
-Effects start running **immediately** after instantiation to ensure they are listening for all relevant actions as soon as possible. Services used in root-level effects are **not** recommended to be used with services that are used with the `APP_INITIALIZER` token.
-
-</div>
-
-## Registering Feature Effects
-
-For feature modules, register your effects by adding the `EffectsModule.forFeature()` method in the `imports` array of your `NgModule`.
-
-<code-example header="admin.module.ts">
-import { NgModule } from '@angular/core';
-import { EffectsModule } from '@ngrx/effects';
-
-import { MoviesEffects } from './effects/movies.effects';
-import * as actorsEffects from './effects/actors.effects';
-
-@NgModule({
-  imports: [
-    EffectsModule.forFeature(MoviesEffects, actorsEffects)
-  ],
-})
-export class MovieModule {}
-</code-example>
-
-### Using the Standalone API
-
-Feature-level effects are registered in the `providers` array of the route config. The same `provideEffects()` function is used in root-level and feature-level effects.
-
-<code-example header="movie-routes.ts">
-import { Route } from '@angular/router';
-import { provideEffects } from '@ngrx/effects';
-
-import { MoviesEffects } from './effects/movies.effects';
-import * as actorsEffects from './effects/actors.effects';
-
-export const routes: Route[] = [
-  {
-    path: 'movies',
-    providers: [
-      provideEffects(MoviesEffects, actorsEffects)
-    ]
-  }
-];
-</code-example>
-
-<div class="alert is-important">
-
-**Note:** Registering an effects class multiple times, either by `forRoot()`, `forFeature()`, or `provideEffects()`, (for example in different lazy loaded features) will not cause the effects to run multiple times. There is no functional difference between effects loaded by `root` and `feature`; the important difference between the functions is that `root` providers sets up the providers required for effects.
```

Comment:
Why is this section removed?

Reply:
I added it back

Reply:
I removed this, and I think it should stay removed.
Looking back it at now it seems like some merges/edits went wrong.

`Registering Effects` shouldn't mention `forRoot` or `forFeature` anymore, but it should show how to register effects using `provideEffects` within different contexts (bootstrapApplication, routes).

I will  go over this section again tomorrow.
Thanks for the review @markostanimirovic